vaadin-card-flow-parent/vaadin-card-flow/src/main/java/com/vaadin/flow/component/card/Card.java

```diff
@@ -85,10 +86,72 @@ public Component getMedia() {
         return SlotUtils.getChildInSlot(this, MEDIA_SLOT_NAME);
     }
 
+    /**
+     * Sets the {@code cardTitle} property. If a {@link #setHeader(Component)
+     * header component} is set, the title will not be displayed. Setting a
+     * title this way removes any title component set using
+     * {@link #setTitle(Component)}. Setting {@code null} or empty string
+     * removes any previously set {@code String} titles.
+     *
+     * @param title
+     *            the title property
+     * @see #setTitle(String, Integer)
+     * @see #setTitleHeadingLevel(Integer)
+     * @see #getTitleAsText()
+     */
+    public void setTitle(String title) {
+        getElement().setProperty("cardTitle", title);
+    }
+
+    /**
+     * Sets the title and the heading level for the title. If a
+     * {@link #setHeader(Component) header component} is set, the title will not
+     * be displayed. Setting a title this way removes any title component set
+     * using {@link #setTitle(String)}. Setting {@code null} or empty title
```

Comment:
Looks like this should refer to `#setTitle(Component)`?

Reply:
Fixed the typo.